firstvoices/backend/search/utils/query_builder_utils.py

```diff
@@ -45,7 +46,12 @@ def get_indices(types):
             indices.add(ELASTICSEARCH_SONG_INDEX)
         elif doc_type == TYPE_STORY:
             indices.add(ELASTICSEARCH_STORY_INDEX)
-        elif doc_type == TYPE_AUDIO or doc_type == TYPE_IMAGE or doc_type == TYPE_VIDEO:
+        elif (
+            doc_type == TYPE_AUDIO
+            or doc_type == TYPE_DOCUMENT
+            or doc_type == TYPE_IMAGE
+            or doc_type == TYPE_VIDEO
```

Comment:
Optional: this is getting to be a long list of ors. Maybe time to convert it to `doc_type in [TYPE_AUDIO...]`